jetstream/export_json.py

```diff
@@ -60,8 +60,9 @@ def _export_table(
         f"""
         SELECT *
         FROM {dataset_id}.{table}
+        WHERE analysis_basis = 'enrollments'
```

Comment:
Experimenter might break if we export exposure based values. Once support has been implemented this should be removed. I'll open a ticket to keep track of this.